examples/express-composition/src/server.ts

```diff
@@ -11,7 +11,7 @@ import path from 'path';
 import {NoteApplication} from './application';
 
 export class ExpressServer {
-  private app: express.Application;
+  public readonly app: express.Application;
```

Comment:
Can you update this in the corresponding docs: https://loopback.io/doc/en/lb4/express-with-lb4-rest-tutorial.html?